server/helpers/requests.ts

```diff
@@ -194,6 +203,39 @@ async function findLatestRedirection (url: string, options: PeerTubeRequestOptio
   return url
 }
 
+function makePOSTAPRequest (url: string, body: any, httpSignature: any, headers: any) {
+  const options = {
+    method: 'POST' as 'POST',
+    json: body,
+    httpSignature,
+    headers
+  }
+
+  return doRequest(url, options)
+}
+
+async function makeFollowRequest (to: { url: string }, by: { url: string, privateKey }) {
```

Comment:
This is only used by test files, so prefer to put it in something like `server/tests/shared`